### DISCUSSION TITLE
Undefined Symbol: When using Third Party Package

### DISCUSSION THREAD

**Discussion by ghost** [skiptools/skip#282] [Q&A]
Hi,

I am new to Skip and wanted to check whether Skip meets the requirements of our existing App. Therefore we need to include the ThirdParty package MapboxMaps. However I receive following errors:  

![image](https://github.com/user-attachments/assets/c214b897-7f24-40ed-9bf0-e4bc3c56ad8f)

To be sure. I ran the same code in a non-Skip project, which build without any problems.

`import SwiftUI

#if SKIP || !os(iOS)
// Skip-specific (e.g., Android) code
#else
import MapboxMaps
#endif

public struct ContentView: View {

    public init() {
    }

    public var body: some View {
        #if SKIP || !os(iOS)
            Text("Test")
        #else
          Map()
        #endif
        
    }
}
` Any help is very much appreciated. Thank you in advance :))

**Comment by DanielSmith1239** [comment]
I'm having the same issue

**Reply by aabewhite** [reply]
This is typically a SwiftPM issue rather than Skip per se. Do you have the maps dependency in your Package.swift? Can you please post it?

**Reply by DanielSmith1239** [reply]
I'm having the issue with other third party packages, not this one in particular.

**Reply by aabewhite** [reply]
It's all the same cause though. Your code is built as a SwiftPM package, so you need to add dependencies to Package.swift rather just through Xcode
